Packs/iDefense/Integrations/FeediDefense/FeediDefense.py

```diff
@@ -0,0 +1,161 @@
+from typing import Dict, Union
+from CommonServerPython import *
+from JSONFeedApiModule import *  # noqa: E402
+
+GLOBAL_INTEGRATION_CONTEXT: dict = {}  # Global variable in order to hold one dictionary that manage all indicator types
```

Comment:
this still has to be global because you call `custom_build_iterator` a few times? or can it be in the func?